nwb_conversion_tools/gui/classes/forms_basic.py

```diff
@@ -279,9 +302,20 @@ def make_forms(self):
             # String types
             if field['type'] == 'str':
                 form = QLineEdit('')
+                if field['required']:
+                    form.setPlaceholderText(field['default'])
```

Comment:
I don't understand these lines. First of all, they seem to be copy/pasted all over the place, which is bad. Second of all, am I reading this right that we are not inputing the data for required fields? Why not?

Reply:
it is copy pasted twice, one for the  class `BasicFormCollaspsible`, and other for `BasicFormFixed`. It does not fill in any value for the required fields, as we had agreed on, it is for the user to input. When using `QLineEdit.text()` (within `read_fields()` method) to read the value of field to copy to dictionary it reads an empty string (if the user did not fill it)

Reply:
if there is a default value specified in the yaml file, that should be used.